cmd/git-sync/main.go

```diff
@@ -144,6 +148,11 @@ var (
 		Help: "How many git syncs completed, partitioned by state (success, error, noop)",
 	}, []string{"status"})
 
+	hookRunCount = prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
This is combined Webhook and Cmdhook, right?  The thinking, I guess, is that most people won't do both?

Reply:
The help should b e clear that this is across all hook types

Reply:
yes it combines hooks under metric `git_sync_hook_run_count_total` , this has a benefit over having two metric  i guess 
you can handle both cases with just alert something like below

`rate(git_sync_hook_run_count_total{status="error"}[5m]) > 0`

but ofc we can devide it two like

```
git_sync_webhook_run_count_total
git_sync_cmdhook_run_count_total
```
---------

"How many hook runs completed, partitioned by name and state "

accually by saying "by name and state" , i thought it is enough as metrics has labels like below, but apparently not:<

```
git_sync_hook_run_count_total{name="cmdhook",status="error"} 5
git_sync_hook_run_count_total{name="webhook",status="error"} 2
```

i think we can think about description , after deciding whether we are going to devide the metric or use labels ?

Reply:
I missed that it was partitioned by name.  That seems fine to me.